examples/src/bin/simple_0rtt_client.rs

```diff
@@ -2,12 +2,13 @@ use std::io::{BufRead, BufReader, Write};
 use std::net::TcpStream;
 use std::sync::Arc;
 
+use pki_types::ServerName;
 use rustls::RootCertStore;
```

Comment:
nitpick: I think there's conventionally whitespace between the crate imports and dependency imports.

Reply:
Good catch, fixed.